toolchain/autoupdate_testdata.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+
+# We run the autoupdate scripts serially, but can build them all at once.
+bazel build -c opt \
+  //toolchain/codegen:codegen_file_test \
+  //toolchain/driver:driver_file_test \
+  //toolchain/lexer:lexer_file_test \
+  //toolchain/lowering:lowering_file_test \
+  //toolchain/parser:parse_tree_file_test \
+  //toolchain/semantics:semantics_file_test
+
+$(dirname %~dp0)/codegen/autoupdate_testdata.sh
+$(dirname %~dp0)/driver/autoupdate_testdata.sh
+$(dirname %~dp0)/lexer/autoupdate_testdata.sh
+$(dirname %~dp0)/lowering/autoupdate_testdata.sh
+$(dirname %~dp0)/parser/autoupdate_testdata.sh
+$(dirname %~dp0)/semantics/autoupdate_testdata.sh
```

Comment:
> Rather than having all of these separate but largely redundant scripts, what about just having this one?

I'd been thinking it was helpful to keep these scripts for people who are used to the current setup, but done.

> If its important to just autoupdate one part of the toolchain, could just accept a positional parameter and dispatch on that here and I think it'd be a bit cleaner than all the one-line test scripts that have to be kept in sync.

For reference, this is going to be the typical use-case, that someone is working on one part of the toolchain, and that changes there don't affect everything. From a development perspective this will help quick iteration.